src/Response/Mapper/ResponseMapperInterface.php

```diff
@@ -2,7 +2,9 @@
 
 namespace DoclerLabs\ApiClientBase\Response\Mapper;
 
+use DoclerLabs\ApiClientBase\Response\Response;
+
 interface ResponseMapperInterface
 {
-    public function map(array $response);
+    public function map(Response $responseData);
```

Comment:
Small thing, but variable name is still responseData.

Reply:
You're right, change done